src/components/Criteria.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import * as Actions from '../actions/assessment';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router'
+
+class Criteria extends Component {
+
+  handleCheck(e) {
+    e.preventDefault();
+    this.props.updateAssesment(this.props.assessment.assessmentObject, this.props.id)
+  }
+
+  render() {
+    return(
+      <tr>
+        <td>
+          <div className="checkbox">
+            <label>
+              <input type="checkbox" name="optionsCheckboxes"
+                checked={this.props.answer}
```

Comment:
if you have to access this.props more than once, I'd recommend to deconstruct it at the top of the render like this:

```
const { answer, text } = this.props;
```

and then just use the variables in the JSX.